src/neural_de.jl

```diff
@@ -51,7 +51,7 @@ end
 Flux.@functor NeuralODE
 
 function (n::NeuralODE)(x,p=n.p)
-    dudt_(u,p,t) = n.re(p)(u)
+    dudt_(u,p,t) = n.re(p)(u, p)
```

Comment:
This isn't correct since Flux models are one-argument functions.

Reply:
But 
```
m = FastChain(FastDense(3, 9, tanh))
```
`m` here is a two argument function right? And that's the input being passed to `NeuralODE`. So I made `rhs2` a two argument function as well.

Reply:
Note that `n.re(p)` returns `rhs2`